packages/termux-keyring/build.sh

```diff
@@ -1,20 +1,40 @@
 TERMUX_PKG_HOMEPAGE=https://github.com/termux
 TERMUX_PKG_DESCRIPTION="GPG public keys for the official Termux repositories"
 TERMUX_PKG_LICENSE="Apache-2.0"
-TERMUX_PKG_VERSION=1.5
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=3.12
```

Comment:
Why are we bumping this from `1.5` to `3.12`?
That version number doesn't immediately make sense to me.

Reply:
Consistency with master branch as commented above.